src/plot_designmatrix.jl

```diff
@@ -59,15 +59,33 @@ function plot_designmatrix!(
             @warn "Sorting does not make sense for time-expanded designmatrices. sort_data has been set to `false`"
             sort_data = false
         end
-        designmat = Matrix(designmat[end÷2-2000:end÷2+2000, :])
+        designmat = Matrix(designmat[end÷2-2000:end÷2+2000, :]) # needs a size(designmat) of at least 4000 x Any
```

Comment:
Yeah I actually ran into this bug last week... Now I understand why the plot didn't work :-D